subworkflows/local/utils_nfcore_mag_pipeline/main.nf

```diff
@@ -298,11 +297,19 @@ def validateInputParameters(hybrid) {
         if (params.busco_db) {
             error("[nf-core/mag] ERROR: Both --skip_binqc and --busco_db are specified! Invalid combination, please specify either --skip_binqc or --binqc_tool 'busco' with --busco_db.")
         }
-        if (params.busco_auto_lineage_prok) {
-            error("[nf-core/mag] ERROR: Both --skip_binqc and --busco_auto_lineage_prok are specified! Invalid combination, please specify either --skip_binqc or --binqc_tool 'busco' with --busco_auto_lineage_prok.")
+    }
+
+    if (!params.skip_binqc && params.binqc_tool == 'busco' && params.busco_db && !params.busco_db_lineage) {
+        log.warn('[nf-core/mag]: WARNING: You have supplied a --busco_db directory, but not specified a lineage with `--busco_db_lineage` - this may cause BUSCO to attempt to auto-download files for you!')
```

Comment:
If `--busco_db` is provided, we are now forcing offline - so this isn't the case any more?

Can we make it so that `--busco_db_lineage` is a required param with default 'auto'? Then have:

```suggestion
        log.warn('[nf-core/mag]: WARNING: You have supplied a database to --busco_db - BUSCO will run in offline mode. Please note that BUSCO may fail if you have an incomplete database and are running with --busco_db_lineage auto!')
```